src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -43,8 +43,11 @@ export const SourceSelectMenu = ({
   onSelect,
 }: ElementProps) => {
   const { walletType } = useAccounts();
-  const { CCTPWithdrawalOnly, CCTPDepositOnly } = useEnvFeatures();
+  // eslint-disable-next-line prefer-const
+  let { CCTPWithdrawalOnly, CCTPDepositOnly } = useEnvFeatures();
 
+  // Only CCTP deposits are supported for Phantom / Solana
+  if (walletType === WalletType.Phantom) CCTPDepositOnly = true;
```

Comment:
[suggestion] would prefer to keep the `const` in there
```
const { CCTPWithdrawalOnly, CCTPDepositOnly: initialCCTPDepositValue } = useEnvFeatures();

const CCTPDepositOnly = walletType === WalletType.Phantom ? true : initialCCTPDepositValue;
```

Reply:
✅